packages/integration-tests/src/process-controller/process_controller.ts

```diff
@@ -159,3 +164,59 @@ export const cdkCli = (
     env: options?.env,
   });
 };
+
+/**
+ *  runWithPackageManager - Factory function that returns a ProcessController for the specified package manager
+ */
```

Comment:
The difference between this function and the one below is very subtle. I'd fill out these JSDocs a little more to explain the difference

Reply:
Done 4c4069bcb4